File: Basic_Algorithms/005_sum_while/sum_while.cpp
#include <iostream>

int main(){

  // video example

  int x = 1, y = 0;
  
  while(x < 11){
    y += x;
    x += 1;
  }

  std::cout << x << " and " << y << std::endl;

  // review question 5
  int a = 1, i = 0;

  while(i < 4){
    a *= 2;
    i++;
  }

  std::cout << a << std::endl;

  // review question 6
  // this will cause inifnity loop
  
  /*
  int c = 1, d = 0;
  
  while(c > 0){
    d += c;
    c++;
  }
  */
}

File: Basic_Algorithms/005_sum_while/sum_while.py
# video example
x = 1
y = 0

while x < 11:
    y += x
    x += 1

print(f'{x} and {y}.')

# review question 5

a = 1
i = 0

while i < 4:
    a *= 2
    i += 1

print(a)

# review question 6
# this will cause infinity loop
'''
b = 1
c = 0

while c > 0:
    b += c
    c += 1

print(b)
'''
